app/src/main/java/com/epfl/drawyourpath/database/FireDatabase.kt

```diff
@@ -8,12 +11,21 @@ import java.util.concurrent.CompletableFuture
 
 private val TIMEOUT_SERVER_REQUEST: Long = 10
 
+
+/**
+ * The Firebase contains files:
+ * -usernameToUserId: that link the username to a unique userId
+ * -users: that contains users based on the UserModel defined by their userId
+ */
 class FireDatabase : Database() {
     val database: DatabaseReference = Firebase.database.reference
+    private val userAuth: User? = FirebaseAuth.getUser()
+    private val usernameToUserIdFileName: String = "usernameToUserId"
```

Comment:
Should be extracted as constants and follow casing convention of constants (all caps).